setup.py

```diff
@@ -38,9 +38,13 @@
     pl_str = "pennylane="
     pl_str_length = len(pl_str)
     pl_version = [line[pl_str_length:].strip() for line in lines if pl_str in line][0]
+    lightning_str = "lightning="
+    lightning_str_length = len(pl_str)
+    lightning_version = [line[lightning_str_length:].strip() for line in lines if lightning_str in line][0]
 
 requirements = [
     f"pennylane @ git+https://github.com/pennylaneai/pennylane@{pl_version}",
+    f"pennylane-lightning @ git+https://github.com/pennylaneai/pennylane-lightning@{lightning_version}",
```

Comment:
Are we sure that `pip` install pennylane-lightning _after_ the `pennylane`? Can PL overwrite PL-lightning if the order happens to be inversed?

Reply:
Hmm good question, I don't know if pip enforces an order here. I think it collects all dependencies recursively first, then attempts to match the version restrictions, before installing each package individually. So I don't think the order in this list should matter.

Reply:
Yes, I agree. Indeed, the race is not happening inside pip. It is between pip and our `make runtime` command. `make runtime` happens before `make frontend`, so I think we need to check this place actually.